tests/openai/test_openai_model_export.py

```diff
@@ -397,3 +397,44 @@ def test_auto_request_retry_is_disabled_when_env_var_is_false(tmp_path, monkeypa
             loaded_model.predict(None)
 
     assert mock_request.call_count == 1
+
+
+def test_embeddings(tmp_path):
```

Comment:
Can we add a mock pyfunc_serve_and_score usage test to validate as well?

Reply:
Sure!